.sync/workflows/leaf/codeql-platform.yml

```diff
@@ -159,11 +159,21 @@ jobs:
         import os
         import requests
         import sys
+        import time
+
+        def get_response_with_retries(url, retries=5, wait_time=10):
+          for attempt in range(retries):
+            response = requests.get(url)
+            if response.status_code == 200:

```

Comment:
technically any code in the 200 range is a success. Do we need to account for any of those?

Reply:
In this case for a GET request the response code is defined as `200`.

- [General REST API HTTP Status Codes](https://gist.github.com/subfuzion/669dfae1d1a27de83e69)
- [Get the combined status for a specific reference](https://docs.github.com/en/rest/commits/statuses?apiVersion=2022-11-28#get-the-combined-status-for-a-specific-reference)

While the others are valid, I wouldn't expect them for this request.